### PR TITLE
docs: adding an explainer page and GIF on insight pages  

### LINKED ISSUE #110: Feature: Usage gifs
### Type of feature

🍕 Feature

### Current behavior

There is no guide on how to use the product and we need the following features explained via gif.

I will work on this for the 100daysofOSS

### Suggested solution

- [ ] Highlights
- [ ] Collaborations
- [ ] Insight Pages

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [x] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by CBID2**
## Description
This PR adds an explainer page and GIF about the insights page. This 
would help new users better understand how to use this feature on OpenSauced.  

## What type of PR is this? (check all applicable)

- [X] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Closes #110 
## Added tests?

- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [X] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: docs/community/insight-pages.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+id: insight-pages
+title: 'View your favorite projects in the Insight pages'
+sidebar_label: 'View your favorite projects in the Insight pages'
+keywords:
+- insight-pages
+---
+
+You know how you see the delicious looking images of pizza pies in the menu when you first walk in a pizza shop? Well, we wanted to replicate this feeling with our Insight pages. This feature is where you pick and choose the repositories that you often contribute to. It's a great way to stay updated on their growth.
```
Comment (bdougie) [inline docs/community/insight-pages.md:9]: The tone is nice, but will be hard to translate the first sentence if we end up doing that in the future. Probably just go with the last sentence.

File: static/gif/insight-pages-demo.gif
Comment (bdougie) [inline static/gif/insight-pages-demo.gif:1]: The gif size 6mb. The other gifs are around 70kb. We will need a smaller gif. I also suggest short short in duration. You can cut out frames and typing to make it smaller and faster.

**Review by bdougie — CHANGES_REQUESTED** [review]

File: static/gif/insight-pages-demo.gif
Comment (CBID2) [reply static/gif/insight-pages-demo.gif:1]: Ok

Comment (CBID2) [reply static/gif/insight-pages-demo.gif:1]: How's this one @bdougie:
 
insight-page-demo

Comment (bdougie) [reply static/gif/insight-pages-demo.gif:1]: It's ok. I can't tell the size without it being committed. The most impactful 5 seconds are at the end, where you show the user the repository being added. I'd suggest cutting it down to only that moment. There is a lot dead time watching typing at the moment and it is not useful.

Comment (CBID2) [reply static/gif/insight-pages-demo.gif:1]: Thanks @bdougie. The size is 606KB

**Comment by bdougie** [comment]
@CBID2 can you add the gif back to the PR? We can move forward with this.

**Comment by CBID2** [comment]
> @CBID2 can you add the gif back to the PR? We can move forward with this.

Done with this and #137 too @bdougie

**Review by bdougie — APPROVED** [review]

**Pull request merged**